topics/assembly/tutorials/vgp_genome_assembly/tutorial.md

```diff
@@ -273,81 +319,81 @@ Meryl will allow us to generate the *k*-mer profile by decomposing the sequencin
->            - "*k-mer size*": `31`
->
->    > <comment-title>Selection of <i>k</i>-mer size</comment-title>
->    >
->    > We used 31 as *k*-mer size, as this length has demonstrated to be sufficiently long that most *k*-mers are not repetitive and is short enough to be more robust to sequencing errors. For very large (haploid size > 10 Gb) and/or very repetitive genomes, larger *k*-mer length is recommended to increase the number of unique *k*-mers.
->    {: .comment}
+>**Step 1**: Run {% tool [Meryl](toolshed.g2.bx.psu.edu/repos/iuc/meryl/meryl/1.3+galaxy6) %} with the following parameters:
+>  1. *"Operation type selector"*: `Count operations`
+>  2. *"Count operations"*: `Count: count the occurrences of canonical k-mers`
+>  3. {% icon param-collection %} *"Input sequences"*: `HiFi_collection (trim)`
+>  4. *"k-mer size selector"*: `Set a k-mer size`
+>  5. "*k-mer size*": `31`
 >
-> 2. Rename it `Collection meryldb`
+> > <comment-title>Selection of <i>k</i>-mer size</comment-title>
+> >
+> > We used 31 as *k*-mer size, as this length has demonstrated to be sufficiently long that most *k*-mers are not repetitive and is short enough to be more robust to sequencing errors. For very large (haploid size > 10 Gb) and/or very repetitive genomes, larger *k*-mer length is recommended to increase the number of unique *k*-mers.
+> {: .comment}
+><br>
+>**Step 2**: Rename output as `meryldb`
 >
-> 3. Run {% tool [Meryl](toolshed.g2.bx.psu.edu/repos/iuc/meryl/meryl/1.3+galaxy6) %} again with the following parameters:
->    - *"Operation type selector"*: `Operations on sets of k-mers`
->        - *"Operations on sets of k-mers"*: `Union-sum: return k-mers that occur in any input, set the count to the sum of the counts`
->        - {% icon param-file %} *"Input meryldb"*: `Collection meryldb`
+>**Step 3**: Run {% tool [Meryl](toolshed.g2.bx.psu.edu/repos/iuc/meryl/meryl/1.3+galaxy6) %} again with the following parameters:
+>  1. *"Operation type selector"*: `Operations on sets of *k*-mers`
+>  2. *"Operations on sets of k-mers"*: `Union-sum: return k-mers that occur in any input, set the count to the sum of the counts`
+>  3. {% icon param-file %} *"Input meryldb"*: `Collection meryldb`
 >
-> 4. Rename it as `Merged meryldb`
+>**Step 4**: Rename it as `Merged meryldb`
 >
-> 5. Run {% tool [Meryl](toolshed.g2.bx.psu.edu/repos/iuc/meryl/meryl/1.3+galaxy6) %} for the third time with the following parameters:
+>**Step 5**: Run {% tool [Meryl](toolshed.g2.bx.psu.edu/repos/iuc/meryl/meryl/1.3+galaxy6) %} for the third time with the following parameters:
 >    - *"Operation type selector"*: `Generate histogram dataset`
 >        - {% icon param-file %} *"Input meryldb"*: `Merged meryldb`
 >
-> 6. Finally, rename it as `Meryldb histogram`.
+>**Step 6**: Finally, rename it as `meryldb histogram`.
 >
 {: .hands_on}
 
+<div id="genomescope">
+<!-- Anchor for linking -->
+</div>
 
 ## Genome profiling with **GenomeScope2**
```

Comment:
you don't need this div, you can just link to `#genome-profiling-with-genomescope2` (found via clicking on the link from the agenda on the published page)

https://training.galaxyproject.org/training-material/topics/assembly/tutorials/vgp_genome_assembly/tutorial.html#genome-profiling-with-genomescope2